pds_doi_service/core/db/doi_database.py

```diff
@@ -456,17 +458,86 @@ def _get_query_criteria_title(v):
     def _get_query_criteria_doi(v):
         return DOIDataBase._get_simple_in_criteria(v, 'doi')
 
+    @staticmethod
+    def _form_query_with_wildcards(column_name, search_tokens):
```

Comment:
This is pretty nice partitioning and setup of the condition clause

Is there any chance that the `search_tokens` will contain dangerous strings like `; drop table *`?

Reply:
The code itself is certainly not doing any *explicit* sanitation for malicious input, but my (rudimentary) understanding was that the usage of "named parameters" (placeholders in the query string and providing a mapping dictionary to `cursor.execute`) was supposed to protect against most(?) injection attacks. However, If a more defensive technique is required, I'm happy to add it.